README.md

```diff
@@ -125,6 +125,29 @@ dashboardTemplates:
 
 Note that you can define more than one dashboard under the `dashboardTemplates` field. 
 
+#### Entity TTL
+
+Entities are deleted by default 8 days after the last time they were indexed or sent out a heartbeat. 
+If this doesn't suit your needs you may also specify your desired behavior under the TTL section of the main `definition.yml` file:  
+
+```yaml
+ttl:
+  # The strategy to follow for the entity deletion. Defaults to AUTOMATIC
+  deletionStrategy: AUTOMATIC
+  # The frequency to delete the entity types. Defaults to EIGHT_DAYS
+  deletionFrequency: EIGHT_DAYS
+```
+
+`deletionStrategy` accepts the values `AUTOMATIC` (default) and `MANUAL`. 
+
+If `AUTOMATIC` is used the entity will be automatically deleted after a certain amount of time, defined by the `deletionFrequency`: 
+- `HOURLY`
+- `DAILY`
+- `EIGHT_DAYS` (default)
+- `QUARTERLY`
+
+If `MANUAL` is used, the `deletionFrequency` field must be left empty or set to `MANUAL`. 
```

Comment:
if the option is not really changing any behaviour it seems simpler to just leave it empty.